File: Project/MyProject/MyProject/Source/MyProject/Private/BasicFrame/Libs/MsgRoute/MsgCmd/LoadedWebResMR.cpp
#include "MyProject.h"
#include "LoadedWebResMR.h"
#include "MsgRouteId.h"

MY_BEGIN_NAMESPACE(MyNS)

LoadedWebResMR::LoadedWebResMR()
	: MsgRouteBase(MsgRouteId::eMRID_LoadedWebRes)
{

}

void LoadedWebResMR::resetDefault()
{
	//m_task = null;
}

MY_END_NAMESPACE

File: Project/MyProject/Tool/CSToolKit/Project/ToolKit/ToolKit/Source/Libs/MsgRoute/MsgCmd/LoadedWebResMR.cs
namespace SDK.Lib
{
    public class LoadedWebResMR : MsgRouteBase
    {
        public ITask mTask;

        public LoadedWebResMR()
        {
            this.setMsgTypeAndsetMsgId(MsgRouteType.eMRT_BASIC, MsgRouteId.eMRID_LoadedWebRes);
        }

        override public void onPutInPool()
        {
            this.mTask = null;
        }
    }
}
